adapters/httpapi_compact.c

```diff
@@ -1530,11 +1544,27 @@ HTTPAPI_RESULT HTTPAPI_CloneOption(const char* optionName, const void* value, co
             result = HTTPAPI_OK;
         }
     }
+    else if (strcmp(OPTION_SET_TLS_RENEGOTIATION, optionName) == 0)
+    {
+        unsigned int* temp = (unsigned int*)malloc(sizeof(unsigned int)); /*shall be freed by HTTPAPIEX*/
```

Comment:
Indeed, it is freed in HTTPAPIEX_Destroy.

Reply:
Fixed on the commit 98666e9b4d46e63d900d4abd1bc1591d3049b5a6